tools/first-address-from-root/.gitignore

```diff
@@ -0,0 +1 @@
+/target
```

Comment:
will we actually have `target` here?

Reply:
no, it's because it was previously not in a workspace and I added by mistake. Good catch, the entire file should be not needed.